ctypes.opam

```diff
@@ -10,7 +10,6 @@ license: "MIT"
 build: [
   [make
     "XEN=%{mirage-xen:enable}%"
-    "COVERAGE=true" {bisect_ppx:installed}
```

Comment:
It looks like this disables coverage checking unconditionally for ctypes builds, enabling coverage only on the test code.

That's not quite what we want: we want to enable coverage checking for the whole build (ctypes code and ctypes tests) when we're building tests, but disable coverage checking altogether when we're not building tests.

Alternatively, since we're not really using the coverage checking at the moment, we could purge it entirely from the code base.